frontend_vue/src/components/ui/CardToken.vue

```diff
@@ -25,7 +26,7 @@
       </div>
       <!--- CTA text --->
       <div
-        class="w-full leading-5 font-semibold border-t-2 border-grey-50 text-grey-700 h-[3rem] rounded-b-xl transition duration-100 hover-card shadow-solid-shadow-grey card-button justify-center items-center flex px-8"
+        class="w-full leading-5 font-semibold border-t-2 border-grey-50 text-grey-700 h-[3rem] rounded-b-2xl transition duration-100 hover-card shadow-solid-shadow-grey card-button justify-center items-center flex px-8"
       >
         {{ isHoverCard ? 'Create Token' : title }}
```

Comment:
i guess this is just github parsing this file weirdly?

Reply:
uhm what's weird about it?